libcxx/test/libcxx/system_reserved_names.gen.py

```diff
@@ -154,6 +154,10 @@
 #define Xp SYSTEM_RESERVED_NAME
 #define Xs SYSTEM_RESERVED_NAME
 
+// These attribute-tokens are not reserved, so the user can macro-define them.
+#define lifetimebound SYSTEM_RESERVED_NAME
+#define noescape SYSTEM_RESERVED_NAME
+
```

Comment:
I wouldn't add these, since we already have a test that would catch this. It's just not enabled in the CI right now because there were some problems.